nomad/structs/csi.go

```diff
@@ -472,18 +472,17 @@ type CSIVolumeGetResponse struct {
 
 // CSIPlugin bundles job and info context for the plugin for clients
```

Comment:
Minor docstring nitpick: we've dropped the job context

Reply:
fixed!